### ISSUE TITLE
Ensures landmarks are unique

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#570]
<dl><dt>Description</dt><dd>Landmarks should have a unique role or role/label/title (i.e. accessible name) combination.

Component's page contains more than one role="region" landmark. 
This issue is common through out the components on website. </dd><dt>Impact</dt><dd>Moderate</dd><dt>Recommendation</dt><dd>The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable.</dd><dt>Source code</dt><dd><pre><code><pre role="region" tabindex="0" class="hljs Code"> </code></pre></dd><dt>Checkpoint</dt><dd>Semantics (1.3.1.a)</dd><dt>URL</dt><dd>NA</dd></dl>

**Comment by schne324** [comment]
best practice - having the codeblocks named from content seems sufficient (along with contextual headings)
